code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -401,3 +401,18 @@
 
 	R.make_shell(src)
 	return TRUE
+
+/obj/item/borg/upgrade/traitorsec
+	name = "Bloodhound upgrade module"
+	desc = "Ilegal upgrade board that unlocks the banned security module."
+	icon_state = "cyborg_upgrade3"
+	require_module = 1

```

Comment:
This simply makes the borg have to choose a module before installing the upgrade. That var should not be needed, since this upgrade is in of itself a module changer.